### DISCUSSION TITLE
Why no Windows Instructions?

### DISCUSSION THREAD

**Discussion by ceb1974** [kionsoftware/kion-cli#64] [Q&A] [answered]
Build instructions are for MacOS and Linux, but no Windows. What are the instructions for Windows?

**Comment by codybuell** [comment, marked as answer]
Hi @ceb1974! Short answer is there hasn't been an ask for it yet and I don't have a Windows system to test on. Since you're asking I'm assuming you have a Windows machine and may be able to help us out. It's a standard Go application so the manual build instructions should work the same as every other OS:

1. [Install Go onto you machine](https://go.dev/doc/install).
2. Clone this repository.
    ```bash
    git clone https://github.com/kionsoftware/kion-cli.git
    ```
3. Build the application.
    ```bash
    go build -ldflags "-X main.kionCliVersion=$(cat VERSION.md)" -o kion.exe
    ```
4. Place it in your path.

Let me know if those steps work for you and we can look at getting the instructions into the readme.

**Reply by ceb1974** [reply]
This works fine, just ended up having to rename the file to "kion.exe".
